backend/database/objects.py

```diff
@@ -312,6 +312,9 @@ def validate_code(self, key, value):
             return value[:max_len]
         return value
 
+    def count_query():
+        return "SELECT n_live_tup FROM pg_stat_all_tables WHERE relname = '%s'" % (Game.__table__.name)
```

Comment:
I'm not a fan of this way if inserting the table names we should be able to do this in SQL alchemy without  the need of a custom string.

Reply:
Totally agree, I'm really not a fan of this. I'll take another crack at it.